profiles/common/modules/features/nexteuropa_feature_set/multisite_drupal_features_set_standard/multisite_drupal_features_set_standard.module

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @file
+ * Declare api for the feature list of standard profile.
+ */
+
+/**
+ * Implements hook_feature_set_api().
+ */
+function multisite_drupal_features_set_standard_feature_set_api() {
+  return array('version' => '1.0');
+}
+
+/**
+ * Implements hook_permission().
+ */
+function multisite_drupal_features_set_standard_permission() {
```

Comment:
Yes it is necessary to have a permission to enable feature sets, and grant it to cem. It makes more sense to do this in a loop instead of adding permissions separately.